hydro_lang/src/ir.rs

```diff
@@ -49,9 +51,180 @@ impl ToTokens for DebugExpr {
     }
 }
 
-impl Debug for DebugExpr {
```

Comment:
We should keep the original `Debug` (this is needed for snapshot testing). Instead, we can put the `Display` behind the `#[cfg(feature = "viz")]` feature flag (and only depend on `prettyplease` from the `deploy` feature or `viz` feature).

Reply:
reinstated